src/index.ts

```diff
@@ -1,4 +1,5 @@
-export {
-    Fuqu,
-    FuquType,
-} from './queue/Fuqu';
+import { fuQuMemory } from './lib/adapters/memory';
+import { fuQuPubSub } from './lib/adapters/pubsub';
+import { fuQuRabbit } from './lib/adapters/rabbit';
+
+export { fuQuPubSub, fuQuRabbit, fuQuMemory };
```

Comment:
I think it would make sense if there was a single funciton, wich would be cool, but I thought I would make that in seperate PR.
What is your idea for the names currently `createXQueue` is shortsighted if we adapt subscriber model  with multiple listeners. `createXAdapter` is confusing, since `adapter` is used in antoher  context in the lib. I am open to suggestions, but I don't think that current naming is a priori bad. 🐱

Reply:
I think `createXQueue` is a perfect fit, even for multiple subscribers. If you are on a train station, there is a single queue and multiple cashiers. And it's still a queue.

My main point is not to have name of the package - fuqu - in a name of exported variable.

Even calling these variables just `pubSub`, `rabbit` makes more sense in context of the package.

Reply:
:cool: I think `createXQueue` is more descriptive. So `createPubSubQueue`, `createRabbitQueue`, `createMemoryQueue`, ok?